packages/admin-panel/src/utils/constants.js

```diff
@@ -0,0 +1,18 @@
+const userStatus = {
```

Comment:
nit-pick: Wrap this objects with `object.freeze` so no one can update its values

example:
```js
const userStatus = Object.freeze({
  VERIFIED: 'verified',
  ACTIVE: 'active',
  DELETE: 'delete',
  BANNED: 'banned',
});

const readableStatus = Object.freeze({
  VERIFIED: 'Verificado',
  ACTIVE: 'Activo',
  DELETE: 'Eliminado',
  BANNED: 'Baneado',
});
```